rootfs/rootfs/usr/local/bin/autologin

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Yeah, if you look at the `Dockerfile` changes in this PR, it used to be written via `echo` in the `Dockerfile` itself, and I simply moved it to an explicit file in the repo and added `exec` in front of that `/bin/login` so that the extra `/bin/sh` process doesn't stick around for every `getty`. :+1: